segway/task.py

```diff
@@ -278,7 +279,7 @@ def divide_posterior_array(posterior_code, num_frames, num_sublabels):
     provide the find_segment_starts() function with data in the same format
     as during the viterbi task.
     """
-    res = zeros((2, num_frames), DTYPE_IDENTIFY)
+    res = zeros((2, num_frames), DTYPE_POSTERIOR)
```

Comment:
This is *probably* correct but I don't understand why this is part of this PR? If it looks like a bug unrelated to these changes it should be a separate PR or have an issue filed.

Reply:
The Viterbi functions have been modified to use string labels, for CNVway. Posterior functions are not modified so only use numeric labels. I had to split the `DTYPE_IDENTIFY` constant (denoting Numpy `int_`) used for both Viterbi and Posterior functions into a `DTYPE_VITERBI` constant (denoting Numpy`str_`) and `DTYPE_POSTERIOR` constant (denoting the Numpy `int_`). Does that explain the change?